src/scaffolding/wizard/OpenStartPageStep.ts

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardExecuteStep } from "vscode-azureextensionui";
+import { openStartPageAfterExtensionUpdate } from "../../commands/startPage/openStartPage";
+import { ScaffoldingWizardContext } from "./ScaffoldingWizardContext";
+
+export class OpenStartPageStep extends AzureWizardExecuteStep<ScaffoldingWizardContext> {
+    public constructor(public readonly priority: number) {
```

Comment:
Is `priority` used for anything?

Reply:
It is used by the wizard framework to order the steps, and is an abstract member, so TS is unhappy if it is not there.